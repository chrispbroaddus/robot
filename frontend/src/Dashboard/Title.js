import React from 'react'
import styled from 'styled-components'
import ZippyImg from '../images/zippy.png'

const Container = styled.div`
  display: flex;
  flex-direction: row;
  align-items: center;
`

const ZippyLogo = styled.img`
  max-width: 69px;
  margin: 20px 40px;
`

const HeaderTitle = styled.h2`
  color: #90a4ae;
  font-family: 'Montserrat-Light';
  font-size: 18px;
`

const Title = props => {
  return (
    <Container>
      <ZippyLogo src={ZippyImg} alt="zippy-robot" />
      <HeaderTitle>Vehicle Dashboard</HeaderTitle>
    </Container>
  )
}

export default Title
