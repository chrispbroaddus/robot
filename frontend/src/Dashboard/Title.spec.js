import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Title from './Title'

describe('Title', () => {
  let props

  beforeEach(() => {
    props = {}
  })

  it('should render self and subcomponents', () => {
    const wrapper = shallow(<Title {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
