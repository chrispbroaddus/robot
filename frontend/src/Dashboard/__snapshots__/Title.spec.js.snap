// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render self and subcomponents 1`] = `
<styled.div>
  <styled.img
    alt="zippy-robot"
    src="zippy.png"
  />
  <styled.h2>
    Vehicle Dashboard
  </styled.h2>
</styled.div>
`;
